roles/base_setup/templates/bash.bashrc.j2

```diff
@@ -58,5 +58,3 @@ if [ -x /usr/lib/command-not-found -o -x /usr/share/command-not-found/command-no
 		fi
 	}
 fi
-
-export LIBVIRT_DEFAULT_URI=qemu:///system
```

Comment:
Is dropping of this variable intentional?

```console
dato@pong:~$ virsh list
 Id   Name   State
--------------------

dato@pong:~$ export LIBVIRT_DEFAULT_URI=qemu:///system

dato@pong:~$ virsh list
 Id   Name     State
------------------------
 1    zaxxon   running
 2    pacman   running
```

Reply:
Ah, good catch, thanks!
I should probably look for a better place for it, but for now I will restore it.